packages/manager/apps/web/client/app/hosting/database/order/public/hosting-database-order-public.controller.js

```diff
@@ -0,0 +1,53 @@
+import { PRODUCT_NAME } from './hosting-database-order-public.constants';
+
+import HostingDatabaseOrderPublic from './hosting-database-order-public.service';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS,
+    OVH_MANAGER_PRODUCT_OFFERS_WORKFLOW_CONSTANTS,
+    HostingDatabaseOrderPublicService,
+  ) {
+    this.OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS = OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS;
+    this.OVH_MANAGER_PRODUCT_OFFERS_WORKFLOW_CONSTANTS = OVH_MANAGER_PRODUCT_OFFERS_WORKFLOW_CONSTANTS;
+    this.HostingDatabaseOrderPublicService = HostingDatabaseOrderPublicService;
+  }
+
+  $onInit() {
+    this.bindings = {
```

Comment:
I'm not sure there is a need to encapsulate properties

Reply:
yes !